### PR TITLE
 jest-glamor-react docs

### LINKED ISSUE #208: Document how to test components with jest-glamor-react
Problem description:

jest-glamor-react works really well with emotion but we don't have any docs for how to use it.

Suggested solution:

Add a document explaining how to use jest-glamor-react with emotion(adding the serializer and writing a test)

### PR DESCRIPTION
**Pull request by greggb**
**What**: Closes #208﻿ - Add docs for jest-glamor-react

**Checklist**:

- [x] Documentation
- [ ] Tests N/A
- [x] Code complete

I also have a before and after screenshot that could be useful, but with the redesign of the site I wasn't sure where that might fit in. Included below.

snapshots

### CODE REVIEW

**Comment by tkh44** [comment]
I went ahead and added the img to the docs. Thanks Gregg!

**Comment by greggb** [comment]
Sounds good, thanks Kye.

File: docs/testing.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import App from './App';
+
+test('Link renders correctly', () => {
+    const tree = renderer.create(<App />).toJSON();
+    expect(tree).toMatchSnapshot();
+});
+```
+
+### Notes
+It's recommended to set your Jest `testEnvironment` to `jsdom`, but you can mock global browser objects instead.
```
Comment (mitchellhamilton) [inline docs/testing.md]: I haven't checked, does manually mocking global browser objects work with jest-glamor-react?

Comment (greggb) [reply docs/testing.md]: I personally just set my env to jsdom, but the docs say it does. https://github.com/kentcdodds/jest-glamor-react#usage

Comment (mitchellhamilton) [reply docs/testing.md]: Ok 👍

**Pull request merged**